src/localizations/en.json

```diff
@@ -17,6 +17,7 @@
     "search": "Search",
     "add": "Add",
     "remove": "Remove",
+    "undo": "Undo",
```

Comment:
Should we remove the api_delegation.undo? I can see arguments for both keeping it as it is and for removing it

Reply:
Well, they're the same and undo is a such a common thing that it should probably be in the common section anyway. So I'll just remove it from api-delegation